examples/benchmarks/train_resnet_imagenet1k.py

```diff
@@ -0,0 +1,320 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This script demonstrates how to train torchvision ResNet architectures on ImageNet using Composer's Trainer. The
+default settings replicate our baseline results. Below illustrates how some arguments can be set to replicate
+our fastest ResNet recipe, our highest accuracy ResNet recipe, and several other settings. This script contains several
+additional features such as saving checkpoints, resume training from a saved checkpoint, speed monitor to track
+training time and throughput, and a learning rate monitor to track learning rate.
+
+Single GPU training:
+    python train_resnet_imagenet1k.py /path/to/imagenet
+
+Log experiments to Weights and Biases:
+    python train_resnet_imagenet1k.py /path/to/imagenet --wandb_logger --wandb_entity my_username
+    --wandb_project my_project --wandb_run_name my_run_name
+
+Single/Multi GPU training (infers the number of GPUs available):
+    composer train_resnet_imagenet1k.py /path/to/imagenet
+
+Manually specify number of GPUs to use:
+    composer -n $N_GPUS train_resnet_imagenet1k.py /path/to/imagenet
+
+Mild ResNet recipe for fastest training to ~76.5% accuracy:
+    composer train_resnet_imagenet1k.py /path/to/imagenet --recipe_name mild --train_crop_size 176 --val_crop_size 224
+    --max_duration 36ep  --loss_name binary_cross_entropy
+
+Medium ResNet recipe highest accuracy with similar training time as baseline:
+    composer train_resnet_imagenet1k.py /path/to/imagenet --recipe_name medium --train_crop_size 176 --val_crop_size 224
+    --max_duration 135ep  --loss_name binary_cross_entropy
+
+Spicy ResNet recipe for our most accurate ResNet over a long training schedule:
+    composer train_resnet_imagenet1k.py /path/to/imagenet --recipe_name spicy --train_crop_size 176 --val_crop_size 224
+    --max_duration 270ep  --loss_name binary_cross_entropy
```

Comment:
Can these have a `\` at the end of the first line so that you can copy-paste these into a terminal?

Same for the wandb command above.

Reply:
Moved these to README instead